### ISSUE TITLE
Fading edge effect on the tab sidebar does not apply to entries below (and including) the selected tab

### ISSUE THREAD

**Issue by Smooth-E** [zen-browser/desktop#2516]
### Captchas

- [X] I have read the instructions.
- [X] I have searched existing issues and avoided creating duplicates.
- [X] I am not filing an enhancement request.

### What happened?

There is a so called "fading edge" effect applied at the bottom of the vertical tab pane when there are too much tabs and the list is overflowing. This "fading edge" strip isn't rendered when you scroll past the selected tab.

| The selected tab is below and isn't yet visible | The selected tab is above |
| --- | --- |
| ![Selected tab below](https://github.com/user-attachments/assets/87531581-4cda-4d3c-8b67-c90b4fac2eb5) | ![Selected tab above](https://github.com/user-attachments/assets/0ce6c80c-bc71-4314-bc14-50c736fd4852)
| As you can see, names and icons of the tabs are faded with a smooth gradient | As you can see, name and icon of the tab are just cut out where list's viewport ends |

Also, this may fall into the "enhancement request" category, but there is no such "fading edge" at the top of the tab strip. I think, it would be nice to add it there too.

Checking as not reproducible on Firefox as vertical tab pane is unique to Zen.

### Reproducible?

- [X] I have checked that this issue cannot be reproduced on Mozilla Firefox.

### Version

1.0.1-a.16

### What platform are you seeing the problem on?

Windows

**Comment by salamijason** [comment]
Hey do you still get this visual bug on a22?

**Comment by Smooth-E** [comment]
> Hey do you still get this visual bug on a22?

Yes. Same issue: no fading edge below selected tab on the bottom, no fading edge at all on the top. I am using the default light theme.

**Comment by salamijason** [comment]
Was able to recreate on our end. will be sending to big dev cheff. Thanks for the info!

**Comment by Smooth-E** [comment]
Well. Here's an update: now there is no "fading edge" effect neither when the select tab is above or below the bottom button group. There is also no such effect for the top button group. It is an improvement in terms of consistency, but I think a fading edge effect is a good addition to indicate there are items below or above, and you need to scroll to see them.

Edit: Browser version is `1.6b`.

**Comment by salamijason** [comment]
Looks like the effect was indeed removed so I'll be closing this issue for now. If you would like it to be re-added please head over to the discussions tab. Thank you for the info!
